recipes-core/custom-nftables-rules/files/nftables-production.conf

```diff
@@ -0,0 +1,40 @@
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+define BUILDER_RPC_SRC = 0.0.0.0  #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_SRC = 1.1.1.1 #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_PORT = 1234 #TODO: Replace with actual port
+
+table inet filter {
+    chain input {
+        type filter hook input priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
+
+        # Production mode rules
+        tcp dport 8551 accept comment "Consensus Client Engine API"
+        tcp dport 30303 accept comment "Execution Client P2P"
+        tcp dport { 8545, 8546 } accept comment "Execution Client RPC"
+        tcp dport 27017 accept comment "Searcher Input Channel"
+        tcp dport 8547 accept comment "Builder State Diff Stream"
+    }
+
+    chain output {
+        type filter hook output priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
+
+        # Production mode rules
+        tcp dport 30303 accept comment "Execution Client P2P"
+        tcp dport { 8545, 8546 } accept comment "Execution Client RPC"
```

Comment:
Hm, should we accept EL RPC to anyone external?
I wonder if there's a good way for us to restrict RPC to only between containers

Reply:
I am not sure about that but I guess princess is running an EL node within podman too. Or that was my impression from Angela's [doc](https://www.notion.so/flashbots/BoB-external-tasks-1126b4a0d876804f924dea7b8ce1cfbc?pvs=4#11e6b4a0d876804195becec99d967cfd).
However, this is just a draft and not the final nftable confs. I would require @bakhtin to check them out and provide me with the conf that we usually require from production instances instead.

Reply:
We wanted to open 8545 for the hackathon so we wouldn't have to wait for their EL node inside to sync, but removing this from the spec now.